sdks/python/apache_beam/runners/portability/expansion_service_main.py

```diff
@@ -39,10 +40,14 @@ def main(argv):
   parser.add_argument(
       '-p', '--port', type=int, help='port on which to serve the job api')
   parser.add_argument('--fully_qualified_name_glob', default=None)
+  parser.add_argument('--default_pickler')
```

Comment:
Any reason not setting cloudpickle as a default pickler in the expansion service? cloudpickle is less compatible or perhaps slower than dill in general cases?

Reply:
We could look at changing the default (for the expansion service, and for Python in general) but I think it's best for now to be explicit.

Reply:
Thanks!